### DISCUSSION TITLE
New file detection for language qualifier: flufflepuff

### DISCUSSION THREAD

**Discussion by MichaelHinrichs** [community/community#123515] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Here's a repo that uses flufflepuff. https://github.com/juju2143/flufflepuff
I would like it to be added to the searchable languages. It uses the file extension ".fp".

**Comment by look** [comment, marked as answer]
Hi @MichaelHinrichs, as I mentioned in response to [your other post](https://github.com/orgs/community/discussions/123517), this is handled by the [Linguist](https://github.com/github-linguist/linguist) library. You can [request a new language by filing an issue](https://github.com/github-linguist/linguist/issues/new?template=new_language.md). However, little-used novelty languages are probably less likely to be accepted. You can, however, search by extension with [`path:*.fp`](https://github.com/search?q=repo%3Ajuju2143%2Fflufflepuff%20path%3A*.fp&type=code).

**Reply by MichaelHinrichs** [reply]
[Brainfuck](https://github.com/search?q=language%3Abrainfuck&type=repositories) made it in. This is just like that.

**Reply by look** [reply]
Anyone can suggest a language for inclusion in Linguist. If enough people use Flufflepuff, it will be accepted by Linguist. If not, it won't be. That's their policy.
